File: beginner/1046.py
n1, n2 = input().split(" ")
n1, n2 = int(n1), int(n2)

duracao = 24 - n1
duracao = duracao + n2

if duracao > 24:
	duracao = n2 - n1
	print("O JOGO DUROU",duracao,"HORA(S)")
else:
	print("O JOGO DUROU",duracao,"HORA(S)")

File: beginner/1046.c
# include <stdio.h>

int main(){
	int hora1, hora2, duracao;
	
	scanf("%i", &hora1);
	scanf("%i", &hora2);

	duracao = 24 - hora1;
	duracao = duracao + hora2;

	if(duracao > 24){
		duracao = hora2 - hora1;
	}

	printf("O JOGO DUROU %i HORA(S)\n", duracao);

	return 0;
}
